src/contract-wrappers/v9/EthereumMailerV9WrapperGlobals.ts

```diff
@@ -117,12 +117,11 @@ export class EthereumMailerV9WrapperGlobals {
 	async setIsYlide(
 		mailer: IEVMMailerContractLink,
 		signer: ethers.Signer,
-		from: string,
 		contractAddresses: string[],
 		isSet: boolean[],
 	) {
 		const contract = this.wrapper.cache.getContract(mailer.address, signer);
-		const tx = await contract.setIsYlide(contractAddresses, isSet, { from });
+		const tx = await contract.setIsYlide(contractAddresses, isSet);
```

Comment:
In many places there is {from} argument which doesn't play any role. It is not possible to have one signer and another "from". Therefore removed it in multiple places but not everywhere.